portfolio/src/main/webapp/index.html

```diff
@@ -3,16 +3,49 @@
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Chelsea+Market&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
+    <div id="header">
+        <div id="home" class="headBox"> <a href="index.html"> Home </a> </div>
```

Comment:
i don't think it's necessary to have both an id and class here. It seems like in your css file, you're defining the style in the `headBox` id. 

In general, you should use style a specific element that only appears once and class for common styling across multiple elements.

Reply:
understood!

Reply:
https://drive.google.com/file/d/1-bJbuJ6oBx8d1h7VHj2JpeIIMZy5a4ra/view?usp=sharing